.github/workflows/release.yml

```diff
@@ -58,4 +58,4 @@ jobs:
         push: true
         tags: ${{ steps.meta.outputs.tags }}
         build-args: |
-          IBC_GO_VERSION="${{ github.ref_name }}"
+          IBC_GO_VERSION=${{ github.ref_name }}
```

Comment:
same fix that was already applied in the 4.3.x release branch but was still in main.